[docs] Improve clarity of 2 files to edit and improve learning (#7285)
sveltejs/svelte: site/content/tutorial/01-introduction/05-nested-components/text.md

title: Nested components
---

> It would be impractical to put your entire app in a single component. Instead, we can import components from other files and then use them as though we were including elements.

Reviewer: ```suggestion
It would be impractical to put your entire app in a single component. Instead, we can import components from other files and use them as though we were including elements.
```
Personal preference, but I think this would make it even clearer.

We now present you 2 files in the editor on the right (upper bar) to click on, `App.svelte` and `Nested.svelte`.

> Each component is a reusable self-contained block of code that encapsulates HTML, CSS, and JavaScript that belong together, written into a `.svelte` file.

Reviewer 2: I might tweak the wording of this slightly as the command and "written into" seem a bit unnatural
```suggestion
Each `.svelte` file holds a component that is a reusable self-contained block of code that encapsulates HTML, CSS, and JavaScript that belong together.
```

> NOTE: Remember that in Svelte, an application is composed from one or more components and each component is a reusable self-contained block of code that encapsulates HTML, CSS, and JavaScript that belong together, written into a `.svelte` file.
> 
> We now present you 2 files in the editor on the right (upper bar) to click on, `App.svelte` and `Nested.svelte`.

Reviewer 3: I think this can be shortened to:

```suggestion
Each component is a reusable self-contained block of code that encapsulates HTML, CSS, and JavaScript that belong together, written into a `.svelte` file.
```

without losing much detail.

Let's add a `<script>` tag to `App.svelte` that imports the file (our component) `Nested.svelte` into our app...

```html
<script>
	import Nested from './Nested.svelte';
</script>
```

> ...then add the component `Nested` to use it in the app markup:

Reviewer 4: ```suggestion
...then use component `Nested` in the app markup:
```
